examples/siro_sandbox/server/client_message_manager.py

```diff
@@ -45,3 +45,9 @@ def change_humanoid_position(self, pos: List[float]) -> None:
         Used to synchronize the humanoid position in the client when changing scene.
         """
         self._message["teleportAvatarBasePosition"] = [pos[0], pos[1], pos[2]]
+
+    def signal_episode_change(self) -> None:
+        r"""
+        Signals the client that the episode is being changed during this frame.
+        """
+        self._message["episodeChanged"] = True
```

Comment:
Same naming comment as in the other PR. I feel like the Unity codebase shouldn't need to have RL or ML-specific concepts like "episode".

Reply:
Related: let's document this function to tell folks what we're really telling the client with this message: that we're requesting a transition effect to help convey the scene-change to the user.